opds.py

```diff
@@ -123,13 +123,15 @@ def slugify_feed_title(cls, feed_title):
         slug = re.sub(' {2,}', ' ', slug)
         return unicode('-'.join(slug.split(' ')))
 
-    def __init__(self, base_url, lane=None, include_search=None, prefix=None):
+    def __init__(self, base_url, lane=None, prefix=None, include_search=None,
+                 license_link=None, suppressed_covers=None):
         if not base_url.endswith('/'):
             base_url += '/'
         self.base_url = base_url
         self.prefix = prefix or ''
         self.lane = lane
         self.include_search = include_search
+        self.license_link = license_link
```

Comment:
I'm going to remove it from this pull request, actually. I'll hit you up with the branch has been updated.